lua/vgit/settings/project_diff_preview.lua

```diff
@@ -15,7 +17,7 @@ return Config({
       desc = 'Unstage'
     },
     buffer_reset = {
-      key = 'r',
+      key = 'R',
```

Comment:
Let's keep this as `r` and have `buffer_hunk_reset` be `gr` to avoid mapping conflict with `reset_all` and keymap consistency for hunk commands being prefixed by `g`.

```suggestion
      key = 'r',
```